frontend/server/src/Controllers/Course.php

```diff
@@ -2049,24 +2051,22 @@ public static function apiListSolvedProblems(\OmegaUp\Request $r) : array {
      * Get Problems unsolved by users of a course
      *
      * @param \OmegaUp\Request $r
-     * @return array{status: string, user_problems: array<mixed, array<int, array<string, mixed>>>}
+     * @return array{status: string, user_problems: array<array-key, array{alias: string, title: string, username: string}>[]}
```

Comment:
```suggestion
     * @return array{status: string, user_problems: array<string, array{alias: string, title: string, username: string}>[]}
```

aquí `string` es mejor que `array-key` (que es básicamente `int|string`).

Reply:
Listo